source/community/reactnative/src/models/action-model.js

```diff
@@ -6,7 +6,8 @@ export class BaseActionModel extends BaseModel{
     constructor(payload, parent) {
         super(payload, parent);
         this.title = payload.title;
-        this.iconUrl = payload.iconUrl;
+        this.url = payload.url;
```

Comment:
It is iconUrl for action elements. Please refer to the schema - https://adaptivecards.io/explorer/Action.OpenUrl.html

Reply:
Property should be `icon'. Inside action, we can consume tintable and URL from icon object. Correct ?

Reply:
Yes correct, I have made the changes accordingly. Please check the same